django/econsensus/publicweb/tests/custom_organizations/views_test.py

```diff
@@ -239,3 +239,21 @@ def test_organisation_user_leave_view_lets_user_delete_themself(self):
         self.assertEqual(
             0, OrganizationUser.objects.filter(user=user, organization=org).count())
         self.assertEqual(reverse('organization_list'), response['Location'])
+
+
+class TestCustomOrganizationChangeOwnerView(TestCase):
+    fixtures = ['organizations.json', 'users.json']
+
```

Comment:
Do you need the fixtures? If not then delete them. If you do then rewrite the tests to avoid using them and delete them.